boards/nxp/frdm_mcxn947/frdm_mcxn947.dtsi

```diff
@@ -59,7 +59,7 @@
 	pinctrl-names = "default";
 };
 
-&flexcomm2_lpi2c2 {
+touch_panel_i2c: &flexcomm2_lpi2c2 {
```

Comment:
As is tradition when NXP comes up with a new display format, we need to determine if this is at all a standard pinout- my guess is no, since I've only seen the pinout used by this display on NXP- supplied displays like the LCD PAR. If so, we should name the shield identifiers appropriately- perhaps `nxp_touch_panel_mipi_dbi_i2c`? I don't want to use the name `nxp_touch_panel_parallel_i2c`, because we also have i2c connectors for parallel RGB displays on the RT10xx EVKs. I'm open to other suggestions for this name though.

Reply:
Thanks @danieldegrasse , will update. About the name `nxp_touch_panel_mipi_dbi_i2c`, my only question is, the i2c is not related to mipi_dbi, should we still include mipi_dbi in the name? thanks.

Reply:
I guess my reasoning is that we already use `parallel` for something else in NXP's display shield names (parallel RGB), so I'm not sure what to name these identifiers. I suggested `mipi_dbi_i2c` because to me it represents the interface type  this connector is designed around (MIPI DBI 8080/6800 mode) and the fact this is an I2C sideband touch controller line on that connector. We could also call this `nxp_8080_i2c_touch_panel` or something similar- I just want to avoid ambiguity between this and other display interfaces with a I2C connection included